### PR TITLE
Use class notation

### PR DESCRIPTION
**Pull request by basz**

### CODE REVIEW

File: src/SlmLocale/Strategy/StrategyPluginManager.php [hunk 1]
```diff
@@ -51,28 +51,27 @@ class StrategyPluginManager extends AbstractPluginManager
...
      * {@inheritdoc}
      */
     protected $factories = [
-        'SlmLocale\Strategy\CookieStrategy'             => InvokableFactory::class,
-        'SlmLocale\Strategy\HostStrategy'               => InvokableFactory::class,
-        'SlmLocale\Strategy\HttpAcceptLanguageStrategy' => InvokableFactory::class,
-        'SlmLocale\Strategy\QueryStrategy'              => InvokableFactory::class,
-        'SlmLocale\Strategy\UriPathStrategy'            => InvokableFactory::class,
-        'slmlocalestrategycookiestrategy'               => InvokableFactory::class,
-        'slmlocalestrategyhoststrategy'                 => InvokableFactory::class,
```
Comment (tdutrion) [inline src/SlmLocale/Strategy/StrategyPluginManager.php]: just wondering, aren't these aliases?

File: Module.php [hunk 2]
```diff
@@ -73,7 +74,7 @@ public function onBootstrap(EventInterface $e)
         $app = $e->getApplication();
         $sm  = $app->getServiceManager();
 
-        $detector = $sm->get('SlmLocale\Locale\Detector');
+        $detector = $sm->get(Locale\Detector::class);
```
Comment (svycka) [inline Module.php]: if you add FQDN name then add \  like this `\Locale\Detector::class`

File: src/SlmLocale/Strategy/StrategyPluginManager.php (hunk 1 above)
Comment (svycka) [inline src/SlmLocale/Strategy/StrategyPluginManager.php]: not sure, I think ServiceManager v2 requires them to be here, but maybe test and see for sure

File: Module.php (hunk 2 above)
Comment (basz) [inline Module.php]: fixed

**Pull request merged**